### PR TITLE
Import the PromLabs PromQL compliance tester

### PR DESCRIPTION
**Pull request by juliusv**
This is a pretty minimal import without too many changes, to get it moved before doing anything more.

The first commit imports it completely unchanged, the second one just fixes minimal things to adjust to the new repo location.

### CODE REVIEW

**Comment by juliusv** [comment]
The test failure seems to be unrelated and due to the fact that the whole repo doesn't have a CircleCI config (happens on https://github.com/prometheus/compliance/pull/6 as well).

File: promql/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# PromQL Compliance Tester
+
+The PromQL Compliance Tester is a tool for running comparison tests between native Prometheus and vendor PromQL API implementations.
+
+The tool was first published and described in https://promlabs.com/blog/2020/08/06/comparing-promql-correctness-across-vendors, and some test results have been published at https://promlabs.com/promql-compliance-tests.
```
Comment (RichiH) [inline promql/README.md:5]:
```suggestion
The tool was [first published and described](https://promlabs.com/blog/2020/08/06/comparing-promql-correctness-across-vendors) in August 2020. [Test results have been published](https://promlabs.com/promql-compliance-tests) on 2020-08-06 and 2020-12-01.
```

File: promql/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+It's still early days for the PromQL Compliance Tester. In particular, we would love to add and improve the following points:
+
+* Test instant queries in addition to range queries.
+* Add more variation and configurability to input timestamps.
+* Flesh out a more comprehensive (and less overlapping) set of input test queries.
+* Automate and integrate data loading into different systems.
+* Test more vendor implementations of PromQL.
+* Version test results and make pretty output presentations easier.
+
+**Note:** Many people will be interested in benchmarking performance differences between PromQL implementations. While this is important as well, the PromQL Compliance Tester focuses solely on correctness testing.
```
Comment (RichiH) [inline promql/README.md:53]:
```suggestion
**Note:** Many people will be interested in benchmarking performance differences between PromQL implementations. While this is important as well, the PromQL Compliance Tester focuses solely on correctness testing. Please contact the [maintainers](../MAINTAINERS.md) if you want work on performance testing.
```

**Review by RichiH — APPROVED** [review]

**Comment by juliusv** [comment]
@RichiH I want this to be mainly an import only PR. Let's keep those changes separate.

**Comment by RichiH** [comment]
Fair; I will put them on top.

**Comment by RichiH** [comment]
https://github.com/prometheus/compliance/pull/10

**Pull request merged**